### PR TITLE
Fixed spelling errors

### PR DESCRIPTION
**Pull request by TJScalzo**
(as well as some grammar/punctuation)

### CODE REVIEW

**Comment by Haroenv** [comment]
Haha thanks, I wrote them on purpose, but readable might be better after all. You misunderstood point "3." though; a bullgit member automatically gets assigned your project, not the other way around

**Comment by TJScalzo** [comment]
Okay, I'll leave this pull request open until you decide if you want to leave the jokes or fix them for readability.

File: README.md [hunk 1]
```diff
@@ -2,27 +2,27 @@
...
+Get the amazing pro bullgit project delivered to your emaildoor!!!
 
-comign soon!
+Coming soon!
 
 # How does it work?
 
 1. Get an entrance ticket to the bullgit pro™ (enterprise) lottery
-3. Get picked one of the members to make your pro(ject)
+3. Get picked as one of the members to make your pro(ject)
```
Comment (Haroenv) [inline README.md]: "a member gets picked"

File: README.md [hunk 2]
```diff
@@ -2,27 +2,27 @@
...
 
-Yes of ourse that's possible in lots of ways‼️
+Yes, of course that's possible in lots of ways‼️
 
 5. Add more useful constraints
 6. Order a project
 9. Join bullgit!
 
-# LICEnSE
+# LICENSE
```
Comment (Haroenv) [inline README.md]: keep this misspelled

File: README.md [hunk 3]
```diff
@@ -2,27 +2,27 @@
 
 ![bullgit pro™](https://bullg.it/media/pixels/bullgit-pro.png)
 
-Get amazign pro bullgit project delivered to your emaildoor!!!
+Get the amazing pro bullgit project delivered to your emaildoor!!!
```
Comment (Haroenv) [inline README.md]: keep this as it was

**Comment by TJScalzo** [comment]
I've undone the changes that you suggested, @Haroenv. 

If you don't mind, could you explain why you want to keep the mistakes? Are they just a joke or do they have a different reason for being there?

**Comment by Haroenv** [comment]
They're meant as a joke, to keep the whole thing playful <4

**Pull request merged**